src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -394,7 +395,7 @@ function initializeNetworkFiltering(frb: FunctionsRuntimeBundle): void {
         .filter((v) => v);
       const href = (hrefs.length && hrefs[0]) || "";
 
-      if (href && !history[href]) {
+      if (href && !history[href] && !href.startsWith("http://localhost")) {
```

Comment:
This prevents logs saying you are accessing an unidentified network resource when you're talking to localhost.